src/gpu/nvidia/cudnn_matmul_lt.hpp

```diff
@@ -215,10 +215,9 @@ struct cudnn_matmul_lt_t : public gpu::primitive_t {
         }
 
         status_t create_scale_binary_pd(impl::engine_t *engine, int ARG) {
-            auto scale_md = dnnl_memory_desc();
-            scale_md.ndims = attr()->scales_.get(ARG).ndims_;
-            scale_md.data_type = attr()->scales_.get(ARG).data_type_;
-            scale_md.format_kind = dnnl_blocked;
+            memory_desc_t scale_md;
```

Comment:
Is `scale_md.ndims = ...` missing on purpose here?

Reply:
`ndims` in scales corresponds to the groups feature which is not supported for NVidia backend. Previous code was not correct in terms of that argument usage. Moreover, the memory descriptor would be re-created regardless of the value (which was always 0, btw) based on src memory desc, so the current version simplifies the former one.